coralproject/talk
# Flag detail

## What does this PR do?

Allows you to see more detail about flags in the moderation queue.

## How do I test this PR?

- flag a comment multiple times with different accounts, and different reasons
- view the flagged stream and click "More Detail"
- you should see the username and message about each flag
